_includes/docs/0.4.x/cloud-foundry.html

```diff
@@ -40,6 +40,11 @@
         <li><pre>$ grunt build</pre></li>
       </ul>
     </li>
+    <li>Run the Gulp Build task to build the optimized JavaScript and CSS files (If you chose Gulp instead of Grunt)
```

Comment:
@codydaig I'm actually going to try to put some work into Gulp this weekend, so let's hold off on removing it if I can make reach parity between Gulp and Grunt.